deprecated/kpca_deprecated.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
```

Comment:
If the original kpca.py script is now deprecated shall we make this deprecated switch compatible with the photoswitch example or would you prefer a modification to the new kpca? (Wherever that might be, I haven't looked for it yet!)

Reply:
You can implement the Morgan fingerprints as a Global_Descriptor class in descriptors/global_descriptors.py